### PR TITLE
Feature/10 main setup page

### PR DESCRIPTION
**Pull request by RagipGashi**
@KlugeCh, please review my issue. Thank you

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # node
 node_modules
-dist
\ No newline at end of file
+dist
+*.js
```
Comment (KlugeCh) [inline .gitignore:4]: insert 'return' -> only prettier problem

File: src/layout-style.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    background-color: white;
+}
+
+div.clearWrap {
+    clear: both;
+}
+
+p.centered {
+    text-align: center;
+}
```
Comment (KlugeCh) [inline src/layout-style.css:73]: insert 'return' -> only prettier problem

File: src/layout.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                <br>
+                <br>
+                <br>
+                <br>
+                <br>
+            </div>
+            <div class="clearWrap"></div>
+        </div>
+    </body>
+</html>
```
Comment (KlugeCh) [inline src/layout.html:73]: insert 'return' -> only prettier problem

**Review by KlugeCh — CHANGES_REQUESTED** [review]
Only minor problems to be fixed.

**Review by RagipGashi — COMMENTED** [review]
Aproved

**Pull request merged**